index.html

```diff
@@ -1185,9 +1185,25 @@ <h2>Extract response headers</h2>
           data-lt="policy origin">origin</a>, <var>policy</var>'s
           <a>subdomains</a> flag is <code>include</code>, and <var>report
           body</var>'s <code>phase</code> property is not <code>dns</code>,
-          abort these steps.
+          return null.
         </li>
 
+        <li>
+          Return <var>report body</var> and <var>policy</var>.
+        </li>
+      </ol>
+
+    </section>
+    <section>
+      <h2 id="deliver-a-network-report" data-dfn-type="dfn"
+          data-export>Deliver a network report</h2>
+      <p>
+      Given a ECMAScript object (<var>report body</var>) and its matching

```

Comment:
To help downstream spec authors understand how this all fits together, it might help to write,

```suggestion
      Given a ECMAScript object (<var>report body</var>, usually returned from <a>Generate a network error report</a> and then augmented by the calling specification) and its matching
```

The NEL spec could also give this kind of guidance to downstream specs in a dedicated section that shows an example of calling these algorithms, but that section doesn't exist yet, and I don't think it's necessary in order to merge this change.

Reply:
done